webapp/src/features/intersections/map/map-component.tsx

```diff
@@ -490,6 +506,8 @@ const IntersectionMap = (props: MAP_PROPS) => {
           sourceData={props.sourceData}
           sourceDataType={props.sourceDataType}
         />
+        <MapLegend />
+        <VisualSettings />
```

Comment:
These can override each other - can we have them toggle or add them to a dynamic grid? This includes the visual settings, information, and legend

Reply:
Added a toggle so that these no longer display over one another